docs/index.md

```diff
@@ -3,12 +3,146 @@
+ * re-run jobs you've created and jobs others have created
+ * download and restore workflows you've created and workflows you've run
+
+The downloaded configurations of storage locations and workflows are in JSON format and can be shared with others.
+
+## Tutorial
+Items needed:
+
+- an orthomosaic image to be processed
+- a GeoJSON file containing the plots of interest
+- an optional YAML file containing [experiment information](https://osf.io/xdkcy/wiki/Configuration%20YAML/) [![external link](img/external_link.png)](https://osf.io/xdkcy/wiki/Configuration%20YAML/)
+
+What we will  be doing:
+
+1. [Upload files and configure](#uploading-and-configuring-files) a server-side file storage definition
+2. Running the [Canopy Cover image-based workflow](#running-a-workflow)
+3. [Downloading the results](#downloading-the-results) to the local computer
+4. [Deleting](#delete-workflow-run) the workflow run
+
+Note that the geographic boundaries of the image don't need to match that of the plots.
+Only those plots that intersect the image will be processed.
+
+Additionally, the coordinate system of the image doesn't need to match that of the plots.
+Plot points are converted automatically to the image's geographic coordinate system.
+
+### Sample data
+
+We also provide an [archive of sample data](https://data.cyverse.org/dav-anon/iplant/projects/aes/cct/diag/sample-data/sample_website_data.tar.gz
+) [![external link](img/external_link.png)](https://data.cyverse.org/dav-anon/iplant/projects/aes/cct/diag/sample-data/sample_website_data.tar.gz) that can be downloaded and used in this Tutorial.
+Be sure to extract the files from this archive before uploading them.
+Uploaded archives are **not** usable by the system.
+
+The following `bash` commands can be used to download the archive and extract the files.
+
+```bash
+curl --output sample_website_data.tar.gz https://data.cyverse.org/dav-anon/iplant/projects/aes/cct/diag/sample-data/sample_website_data.tar.gz
+tar -xzf sample_website_data.tar.gz
+```
+
+### Uploading and configuring files
+
+On the main page of the site, click the `files` link under **Data Sources**.
+This will take you to the file storage definition page
+
+<img src="img/landing_click_files.png" width="450" alt="File storage definition link" aria-label="File storage definition link" title="Click on Files link"/>
+
+Next click the upload icon, select and upload the files to process.
+You may see an <img src="img/configure_files_uploading_files.png" width="90" alt="Uploading files" aria-label="Uploading files" title="Uploading files"/> message on the icon if it takes a few seconds to upload the files.
+You can also drag-and-drop files onto the upload location.
+
+<img src="img/configure_files_upload.png" width="450" alt="Click upload files icon" aria-label="Upload files link" title="Click or drag-and-drop to upload files"/>
+
+Next, select "Server Side" from the drop down on the upper right of the page, and click "New".
```

Comment:
it is unclear what is happening here - 
- after I upload, I don't see a list of the files 
- not clear what 'server side' / new should do - is this just setting the destination? Would it be possible to set the destination at the time of upload?

Reply:
> after I upload, I don't see a list of the files

I added a line about seeing the uploaded files on the next step

> not clear what 'server side' / new should do - is this just setting the destination? Would it be possible to set the destination at the time of upload?

At the start of the section I've added some more text explaining what we're doing. I've also added clarification on the line

Reply:
I think that "Server side" is too jargony. There must be a more common term for what this is

Reply:
Issue written to change this: https://github.com/AgPipeline/issues-and-projects/issues/564